Cargo.toml

```diff
@@ -14,13 +14,15 @@ exclude = [
 ]
 
 [features]
-default = ["std", "spill-stack"]
+default = ["std", "spill-stack", "memoisation"]
 # Integrate with the standard library
 std = []
 # Enable nightly-only features like better compiler diagnostics
 nightly = []
 # Allows deeper recursion by dynamically spilling stack state on to the heap
 spill-stack = ["stacker", "std"]
+# Allows parser memoisation, speeding up heavily back-tracking parsers and allowing left recursion
+memoisation = []
```

Comment:
Maybe we should use the US spelling? Or a different feature name?

Reply:
I'm trying to think of a different name, and can't think of one. I personally don't feel strongly about the spelling, but I wish we could maybe just have two features with the different spellings that require each-other :P